### PR TITLE
Ltd 879 use vcap services

### PR DESCRIPTION
**Pull request by dgnball**
Use the VCAP services from CF rather than Vault for credentials.

### CODE REVIEW

**Review by ejayesh — APPROVED** [review]
This looks good to me

File: dags/export_lite_db.py [hunk 1]
```diff
@@ -8,16 +10,35 @@
 from boto3.session import Session
 
 
+class ANON_VARS:
```
Comment (saruniitr) [inline dags/export_lite_db.py:13]: may be `ANON_ENV_VARS`? as these are environment variables. But it is up to you to make this change.

File: example_vcap.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "relational"
+      ],
+      "instance_name": "lite-anonymise-api-pg",
+      "binding_name": "anonymise-api-db",
+      "credentials": {
+        "uri": "postgres://postgres:password@localhost:5462/lite-api"
+      }
+    }
+  ]
+}
```
Comment (saruniitr) [inline example_vcap.json:34]: new line required

File: .gitignore [hunk 3]
```diff
@@ -73,3 +73,4 @@ spire-local/
 /logs/
 /.venv/
 /.idea/
+vcap.json
```
Comment (saruniitr) [inline .gitignore:76]: newline required

**Review by saruniitr — APPROVED** [review]
few minor comments. lgtm.

**Pull request merged**